internal/repository/inmem/port.go

```diff
@@ -41,31 +41,31 @@ func (p *Port) Copy() *Port {
 	}
 }
 
-func PortToDomain(p *Port) (*domain.Port, error) {
+func portStoreToDomain(p *Port) (*domain.Port, error) {
```

Comment:
you must check that p is not nil, which has the potential of panic for nil pointer exception.

Reply:
right, thank you. Fixed + added unit tests.